src/main/java/com/gg/server/domain/match/service/MatchService.java

```diff
@@ -197,24 +197,26 @@ private void deleteUserFromQueue(UserDto userDto, LocalDateTime startTime) {
 
     /**
      * LIVE, BEFORE 상태인 토너먼트와 진행 시간이 겹치지 않으면 true, 겹치면 false
-     * @param time 현재 시간
+     * @param startTime 현재 시간
      * @return 종료되지 않은 토너먼트 있으면 true, 없으면 false
+     * @throws SlotNotFoundException 현재 시간에 해당하는 슬롯이 없을 경우
      */
-    private boolean isExistTournamentNotEnded(LocalDateTime time) {
+    private boolean isExistTournamentNotEnded(LocalDateTime startTime) {
         List<Tournament> tournamentList = tournamentRepository.findAllByStatusIsNot(TournamentStatus.END);
-        if (tournamentList.isEmpty()) {
-            return false;
-        }
+        int gameInterval = slotManagementRepository.findCurrent(startTime)
+            .orElseThrow(SlotNotFoundException::new)
+            .getGameInterval();
+        LocalDateTime endTime = startTime.plusMinutes(gameInterval);
         for (Tournament tournament : tournamentList) {
-            if (time.isAfter(tournament.getStartTime()) &&
-                time.isBefore(tournament.getEndTime())) {
-                return false;
+            if (startTime.isAfter(tournament.getStartTime()) && startTime.isBefore(tournament.getEndTime()) ||
+                    endTime.isAfter(tournament.getStartTime()) && endTime.isBefore(tournament.getEndTime())) {
```

Comment:
```suggestion
            if ((startTime.isAfter(tournament.getStartTime()) && startTime.isBefore(tournament.getEndTime())) ||
                    (endTime.isAfter(tournament.getStartTime()) && endTime.isBefore(tournament.getEndTime()))) {
```
or 연산 기준으로 괄호로 안묶여있어서 제대로 동작 안하지 않을까요?

Reply:
and 연산자가 우선순위라서 상관 없을 거 같긴한데  가독성 위해 추가할까요?

Reply:
아 그렇군요 동작에 상관없다면 한나님 편하신대로 하면 될것 같아요

Reply:
그렇다면 이대로 가도록 하겠습니다 ㅎㅎ 감사합니당